modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -93,9 +93,9 @@ def render_model_errors
   end
 
   def model_errors_to_json_api
-    errors = @notice_of_disagreement.errors.to_h.map do |source, data|
```

Comment:
Need to call explicit to_h method here too

Reply:
Does this still need to happen?

Reply:
We're now iterating over an array of error objects